hbase_storage/src/main/java/com/splicemachine/si/impl/region/TxnTimeTraveler.java

```diff
@@ -52,14 +52,14 @@ public TxnTimeTraveler(TxnFinder txnFinder) {
             if(middleTS == needleTS) {
                 return middleTx;
             } else if(needleTS < middleTS) {
+                right = toRowKeyWithBucket(middleTx.getFirst(), bucket);
                 if(middleTS == rightTS)
                 {
                     reversed = true;
                     changed = true;
                 }
                 else if(middleTS < rightTS)
                 {
-                    right = toRowKeyWithBucket(middleTx.getFirst(), bucket);
                     rightTS = middleTS;
                     changed = true;
```

Comment:
`changed = true;` is common in both conditional branches, can be moved outside `if`

Reply:
Thanks! I actually saw it after I pushed PRs to 3.0 and 3.1, will fix it later if that's OK.